snuba/utils/threaded_function_delegator.py

```
@@ -84,6 +86,9 @@ def execute_callback() -> None:
                     if self.__callback_func is not None:
                         self.__callback_func(primary_result, other_results)
                 except Exception as error:
-                    logger.exception(error)
+                    if self.__ignore_secondary_exceptions:
```

how does this only ignore exceptions from tiger?

---

oh nvm, I understand now